qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -632,8 +635,23 @@ degradation in performance at the end of the transfer. Especially on large
 shards, this can give a huge performance improvement. 2. The ordering guarantees
 can be `strong`[^ordered], required for some applications.
 
+The `wal_delta` transfer method only transfers the difference between two
+shards. More specifically, it transfers all operations that were missed to the
+target shard. The [WAL] of both shards is used to resolve this. There are two
+benefits: 1. it will be very fast, because it only transfers the difference
+rather than all data. 2. the ordering guarantees can be `strong`[^ordered],
+required for some applications. Two disadvantages are: 1. it can only be used to
+transfer to a shard that already exists on the other node. 2. applicability is
+limited because the WALs normally don't hold more than 64MB of recent
+operations. But that should be enough for a node that quickly restarts, to
+upgrade for example. If a delta cannot be resolved, this method automatically
+falls back to `stream_records` which equals to transferring the full shard.
+
```

Comment:
```suggestion
The `wal_delta` transfer method only transfers the difference between two
shards. More specifically, it transfers all operations that were missed to the
target shard. The [WAL] of both shards is used to resolve this. There are two
benefits: 1. It will be very fast, because it only transfers the difference
rather than all data. 2. The ordering guarantees can be `strong`[^ordered],
required for some applications. Two disadvantages are: 1. It can only be used to
transfer to a shard that already exists on the other node. 2. Applicability is
limited because the WALs normally don't hold more than 64MB of recent
operations. But that should be enough for a node that quickly restarts, to
upgrade for example. If a delta cannot be resolved, this method automatically
falls back to `stream_records` which equals transferring the full shard.

```